### ISSUE TITLE
User migrator check for roles/schema compatibility

### ISSUE THREAD

**Issue by javitonino** [CartoDB/cartodb#12939]
We have discovered that some organizations have schemas and roles for users that no longer exists. Probably the user was deleted but its database and role were not.

In this case, import fails because the roles are not restored. The workaround is to manually create the missing roles (obtain with `\dn` in the source DB, and `CREATE ROLE ...` in the target one).

It would be nice to have a check to verify this before trying to export, so an error can be issued before starting.
